spec/shared/rational/truncate.rb

```diff
@@ -55,7 +55,7 @@
     end
   end
 
-  describe "with an invalid valud for precision" do
+  describe "with an invalid value for precision" do
```

Comment:
Just for reference: this one has been applied upstream as well, there are no other differences between this file and the upstream version.